src/decorators.ts

```diff
@@ -126,6 +126,32 @@ export function ResponseSchema(
         },
       }
 
+      const oldSchema =
+        source.responses[statusCode]?.content[contentType].schema
+
+      const makeNewSchema = () => {
+        if (oldSchema.oneOf) {
+          return {
+            oneOf: _.concat([...oldSchema.oneOf], schema),
```

Comment:
```suggestion
            oneOf: [...oldSchema.oneOf, schema],
```

I think it's a good idea to avoid using lodash unless necessary. Maybe one day we can drop the dependency here altogether.

Reply:
sounds good